packages/webpack-cli/lib/utils/arg-parser.js

```diff
@@ -63,6 +63,12 @@ function argParser(options, args, argsOnly = false, name = '', helpFunction = un
             if (!option.multiple) {
                 // Prevent default behavior for standalone options
                 parserInstance.option(flagsWithType, option.description, option.defaultValue).action(() => {});
+
+                if (option.negative) {
+                    // commander requires explicitly adding the negated version of boolean flags
+                    const negatedFlag = `--no-${option.name}`;
+                    parserInstance.option(negatedFlag, `negates ${option.name}`).action(() => {});
+                }
```

Comment:
Since this check is not limited to any flags of a certain kind why not moving it outside? That way we can also remove the redundancy.

Reply:
done